src/activitylog/activitylog.html

```diff
@@ -14,27 +18,38 @@ <h1>Extension Activity Monitor</h1>
       <div class="notice"></div>
       <div class="activity-log-wrapper">
         <h2>Activity Logs</h2>
-        <div class="activity-log-container">
-          <div class="log-table">
-            <table>
-              <thead>
-                <tr>
-                  <th>Extension Id</th>
-                  <th>Timestamp</th>
-                  <th>API Type</th>
-                  <th>API Name</th>
-                  <th>View Type</th>
-                </tr>
-              </thead>
-              <tbody></tbody>
-            </table>
-          </div>
-          <div class="show-log-detail-wrapper" hidden>
-            <div id="showLogDetails"></div>
-            <div class="close"></div>
-          </div>
-        </div>
+        <log-view></log-view>
       </div>
     </div>
+    <template class="log-table-template">
+      <link rel="stylesheet" href="/lib/web-component/log-view/log-view.css" />
```

Comment:
Remove `/` from `/>`. This is not XHTML, the `/` serves no purpose.

Reply:
It is done by prettier. I can tell prettier to ignore this if you want.
Note: I faced this issue earlier when Luca asked me to [remove `/` self-closing tag meta tags](https://github.com/mozilla/extension-activity-monitor/pull/3#discussion_r443464519). But [he suggested to not fight with prettier for it.](https://github.com/mozilla/extension-activity-monitor/pull/3#discussion_r444405359)

Reply:
Never mind then. Prettier is wrong, but it's not worth the effort.